postgresql/restore.sh

```diff
@@ -6,13 +6,13 @@
 
 # Before running the restore, you'll have to stop the ECS task to close any open connections.
 # Then, run the following to drop the database.
-# psql -U concordia -h $POSTGRES_HOST
+# psql -U concordia -h $POSTGRESQL_HOST
 # \c postgres
 # drop database concordia;
 
-export POSTGRES_HOST=
+export POSTGRESQL_HOST=localhost
```

Comment:
Shouldn't this pass the value through if it's set outside? 
```suggestion
export POSTGRESQL_HOST=${POSTGRESQL_HOST:-localhost}
```